packages/core/src/types.ts

```diff
@@ -33,13 +59,13 @@ export interface CCSSOptions {
      * Map of supported pseudo selectors
      */
     pseudoMap: Partial<CCSSPseudoMap>
-}
 
-export type CCSSInput = any
+    __ccss: CCSSAnyFunction
+}
 
-export type CCSSParser = (input: CCSSInput, string, CCSSOptions, CCSSOriginal?) => CCSSInput
+export type CCSSParser = (input: CCSSInput, prop: string, options: CCSSOptions, original?: CCSSOriginal) => CCSSInput
 
-export type CCSSPipe = (...fn: CCSSParser[]) => CCSSInput
+export type CCSSPipe = (...fn: CCSSParser[]) => CCSSParser
```

Comment:
@wintercounter I've added the return type as `CCSSParser` because I was unable to find anywhere in the code need for having it to return `any` (a.k.a. `CCSSInput`).

Reply:
Yeah, that's good.